src/elements.rs

```diff
@@ -8,6 +8,10 @@ use webdriver::command::WebDriverCommand;
 use webdriver::common::FrameId;
 use webdriver::error::WebDriverError;
 
+/// An element id.
```

Comment:
Is there a part of the spec we can link to for the definition of an element ID?

Also, are there other places in the API where this type may be useful?

Reply:
Kind of. There's [11. Elements](https://www.w3.org/TR/webdriver1/#elements) which lists this as the second item:
```
The web element identifier is the string constant "element-6066-11e4-a52e-4f735466cecf". 
```

Reply:
As for where else it might be useful, I'm not sure. It's definitely useful for constructing an element or any kind of (de)serialization involving elements.

Reply:
Actually, reading that, I _think_ we've actually been using the wrong term. I think in all these instances, these are "web element references" (as defined in the third item), not _the_ "web element identifier" (which is a constant string). I think we should quote the spec directly in the docs here:

> Each element has an associated web element reference that uniquely identifies the element across all browsing contexts. The web element reference for every element representing the same element must be the same. It must be a string, and should be the result of generating a UUID.

And also link to section 11.

Reply:
I also renamed `ElementId` to `ElementRef`